capstone/src/main/java/com/google/step/servlets/RestaurantServlet.java

```diff
@@ -91,21 +80,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Hard coded while we don't have Maps API functionality
     GeoPt geoPoint = new GeoPt((float) 42.23422, (float) -87.234987);
 
-    Entity restaurantInfo = new Entity("RestaurantInfo", id);
-    restaurantInfo.setProperty("restaurantKey", id);
-    restaurantInfo.setProperty("name", name);
-    restaurantInfo.setProperty("email", email);
-    restaurantInfo.setProperty("location", geoPoint);
-    restaurantInfo.setProperty("story", story);
-    restaurantInfo.setProperty("cuisine", cuisineList);
-    restaurantInfo.setProperty("phone", phone);
-    restaurantInfo.setProperty("website", website);
+    // The following value is hardcoded while we implement the properties.
+    String status = "OKAY";
 
-    // Both of the following values are hardcoded while we implement the properties.
-    restaurantInfo.setProperty("score", 2.5);
-    restaurantInfo.setProperty("status", "OKAY");
+    // Dummy value as id will be set in the client using the Entity's id.
+    long id = 0;
```

Comment:
You could probably get rid of the ID field as a constructor parameter and create a setter that sets the ID as a private method in the class.

Reply:
I took it out of the constructor! The setter has to be public I believe in order for RestaurantClient to set it with the Datastore ID.